### ISSUE TITLE
Decoding task links offline possible?

### ISSUE THREAD

**Issue by hyperknot** [vicb/flyxc#124]
Almost all pilots have started using flyxc.app tasks for task declarations for national records, etc. My question is that are these links encoding the whole task, or they just point to a DB record on the backend server?

Basically, what happens if flyxc.app goes offline or disappears. We have lost a lot of task declarations with previous websites, when they went offline all the information was gone. I suspect that flyxc is actually URL encoding the task (the URL changes when I move the points), but I found no resource about decoding them.

Would it be possible to include a small node script in this repository, which would get a url query like:
```
?p=i%7B%7E_HqqvnB%7ClpAznKujBso_CeamA%7C_sB&s=25.0
```
and return the data? As an alternative, the xctrack planner url would also be nice:
http://xctrack.org/xcplanner?route=47.34917%2C18.30697%3A46.93222%2C18.24299%3A46.94945%2C18.90101%3A47.34916%2C18.30694

**Comment by vicb** [comment]
**TL;DR**
You can download tasks as GPX, XCTsk, ... those formats should be easy to share across apps.

`i%7B%7E_HqqvnB%7ClpAznKujBso_CeamA%7C_sB` is actually [a format created by Google](https://developers.google.com/maps/documentation/utilities/polylinealgorithm).

There is a [page](https://developers.google.com/maps/documentation/utilities/polylineutility) that can be used to decode it

**Comment by hyperknot** [comment]
Thanks. I understand it now. One important thing is that the url has to be URL decoded, but afterwards it can be plugged into Google decoder.
